CHANGELOG.md

```diff
@@ -0,0 +1,252 @@
+# Changelog
+
```

Comment:
Having a top-level heading here, will break the include in the docs, so we shouldn't have one.
I guess people can figure out they're looking at a ChangeLog/Release Notes, when looking at `CHANGELOG.md`
```suggestion
```